<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Testing</class>
 <widget class="QWidget" name="Testing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="leDataTime">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>60</y>
     <width>270</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>日期时间</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>801</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lbSampleText1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>50</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>样本号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leSampleNo1">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>600</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>600</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pbExit">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>420</y>
     <width>130</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbLable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>等待温度稳定后开始测量！</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_Bottle1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>1号瓶</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_Bottle2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>110</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>2号瓶</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_Bottle3">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>110</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>3号瓶</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_Bottle4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>4号瓶</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_Bottle5">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>290</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>5号瓶</string>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>260</y>
     <width>800</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_Bottle1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_Bottle1_St">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(223, 223, 223);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_Bottle2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>210</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_Bottle2_St">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>110</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(223, 223, 223);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbSampleText2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>150</y>
     <width>50</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>样本号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leSampleNo2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>150</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_Bottle3">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>210</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_Bottle3_St">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>110</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(223, 223, 223);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbSampleText3">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>150</y>
     <width>50</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>样本号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leSampleNo3">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>150</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_Bottle4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>380</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_Bottle4_St">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>290</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(223, 223, 223);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbSampleText4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>330</y>
     <width>50</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>样本号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leSampleNo4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>330</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_Bottle5">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>380</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_Bottle5_St">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>290</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(223, 223, 223);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbSampleText5">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>330</y>
     <width>50</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>样本号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leSampleNo5">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>330</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_Next">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>360</y>
     <width>130</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>下一组</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTempDW1">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>430</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>℃</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTemp1">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>430</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>温度</string>
   </property>
  </widget>
  <widget class="QLabel" name="leTempDis">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>430</y>
     <width>71</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(223, 223, 223);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lb_Title">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>5</y>
     <width>400</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_Prev">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>290</y>
     <width>130</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>上一组</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
